### PR TITLE
Implement Msg parsing in TxHook

### LINKED ISSUE #149: Implement Msg parsing in TxHook
Template already exists. For now let's just use the transaction input (not looking at events at all). We can change later as needed.

### PR DESCRIPTION
**Pull request by taobun**
fixed #149

### CODE REVIEW

File: scan/src/App.re [hunk 1]
```diff
@@ -2,7 +2,11 @@ module Styles = {
   open Css;
 
   let container =
-    style([width(`percent(100.)), height(`percent(100.)), position(`relative)]);
+    style([
```
Comment (sorawit) [inline scan/src/App.re]: Don't change this. Update line length to 100.

File: scan/src/rpc/TxHook.re [hunk 2]
```diff
@@ -1,14 +1,48 @@
 module Msg = {
   module Send = {
-    // TODO
+    type t = {
+      fromAddress: string,
+      toAddress: string,
+      amount: string,
```
Comment (sorawit) [inline scan/src/rpc/TxHook.re]: Should be float. See `ValidatorHook.Validator.t`

File: scan/src/rpc/TxHook.re [hunk 3]
```diff
@@ -1,14 +1,48 @@
...
+        fromAddress: json |> field("from_address", string),
+        toAddress: json |> field("to_address", string),
+        amount: json |> field("amount", string),
+      };
   };
 
   module Store = {
-    // TODO
+    type t = {
+      code: string,
```
Comment (sorawit) [inline scan/src/rpc/TxHook.re:32]: codeHash?

File: scan/src/rpc/TxHook.re [hunk 4]
```diff
@@ -1,14 +1,48 @@
...
+      owner: string,
+    };
+
+    let decode = json =>
+      JsonUtils.Decode.{
+        // TODO: change to `code` and `owner` after update node
+        code: json |> field("Code", string),
```
Comment (sorawit) [inline scan/src/rpc/TxHook.re]: Changed in #144?

**Review by sorawit — CHANGES_REQUESTED** [review]

File: scan/src/rpc/TxHook.re (hunk 3 above)
Comment (sorawit) [reply scan/src/rpc/TxHook.re:32]: Actually it's code. Nevermind.

Comment (taobun) [reply scan/src/rpc/TxHook.re:32]: code si kub

**Review by sorawit — COMMENTED** [review]
@evilpeach

File: chain/cmd/request/main.go [hunk 5]
```diff
@@ -54,6 +56,14 @@ func main() {
 		panic(err)
 	}
 
+	// Send token
+	to, _ := sdk.AccAddressFromBech32("band13zmknvkq2sj920spz90g4r9zjan8g584x8qalj")
```
Comment (sorawit) [inline chain/cmd/request/main.go]: Remove this?

**Review by sorawit — CHANGES_REQUESTED** [review]

File: chain/cmd/request/main.go (hunk 5 above)
Comment (taobun) [reply chain/cmd/request/main.go]: It's example code, commented already

**Review by evilpeach — APPROVED** [review]
LGTM

**Review by sorawit — APPROVED** [review]

**Pull request merged**